content/tutorials/dsx-subscribe.textile

```diff
@@ -0,0 +1,249 @@
+
+h2. Step 2 - Confirming your subscription using the DSX dashboard
+
+For this tutorial, we'll be making use of the *London tube schedule updates* data stream containing the data provided by "Transport For London (TFL)":https://tfl.gov.uk/. As the name suggests, TFL is a local government body responsible for the transport system in Greater London, England. All transport related data can be obtained for free from their official website. 
+
+p(did-you-know). Most companies make their commercial and open data streams publicly available through the "Ably Hub":/hub, which is a hosted catalog of data streams. The data stream we'll be using in this tutorial is listed as "London tube schedule updates":https://www.ably.io/hub/products/10 on the "Ably Hub":https://www.ably.io/hub/. These products require the owner to confirm access, and others may charge for usage of their product.
+
+Ideally, a producer of certain data would push or publish their data directly into Ably DSX, for others to subscribe to. However, to demonstrate the functionality of DSX, Ably is pulling TFL's data from various public sources and making it available as a realtime stream that anyone can then subscribe to.
+
+The full documentation for the TFL data stream can be found on the "TFL DSX Source page":https://dsx-sources.herokuapp.com/source/tfl.
+
+After you've been added as a subscriber by the data producer, you will be able see that data stream listed on your DSX consumer dashboard. To reach that page, log into your Ably DSX account and select the consumer dashboard as shown below:
+
+<a href="/images/tutorials/dsx-dashboard.png" target="_blank">
+  <img src="/images/tutorials/dsx-dashboard.png" style="width: 100%" alt="DSX Dashboard">
+</a>
+
+To confirm the subscription to the said data stream, switch to the 'Subscriptions' tab, and you should see the data stream listed there. You can also see the message quota, license type and other information related to your subscription to the said data stream.
+
+<a href="/images/tutorials/subscriptions-tab-dsx.png" target="_blank">
+  <img src="/images/tutorials/subscriptions-tab-dsx.png" style="width: 100%" alt="DSX Dashboard">
+</a>
+
+Now that your account has a valid subscription with the product let's go ahead and write some code to start receiving updates.
+
+h2. Step 3 - Buiding the HTML to display results
+
+In this tutorial, we'll be subscribing to arrival updates for London tube lines at certain stations, that you can select from a drop-down menu. Go ahead and paste the following basic HTML in a file and name it @index.html@. Note that each of the options in the drop-down menu corresponds to the respective Naptan Code as illustrated in the "Ably-TFL product documentation":https://dsx-sources.herokuapp.com/source/tfl. The Naptan Code is a combination of @line@ and @Station ID@. Note that we've also added a reference to Ably's JavaScript library, the jQuery library and a @main.js@ file in which we'll include our app logic.
```

Comment:
NAPTAN isn't a combination of line + the ID of the station., it is the ID for the station. The line is just part of the channel name.